<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pymunk симуляция</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutMain" stretch="0,0,0,0,0,0,1,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_bd">
      <item>
       <widget class="QPushButton" name="saveBDbutton">
        <property name="text">
         <string>Сохранить в базу данных</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importBDbutton">
        <property name="text">
         <string>Загрузить из базы данных</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Параметры пространства:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayoutG">
        <item row="0" column="1">
         <widget class="QLabel" name="labelG">
          <property name="text">
           <string>Ускорение свободного падения (g):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="pmGspinBox">
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>1200.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayoutFR">
        <item row="0" column="1">
         <widget class="QLabel" name="labelFR">
          <property name="text">
           <string>Коэффициент трения дна:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="pmFRspinBox">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.800000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Настройки PYGAME</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutPGame">
      <item>
       <layout class="QFormLayout" name="formLayoutPGame">
        <item row="0" column="1">
         <widget class="QLabel" name="labelPgwidth">
          <property name="text">
           <string> Ширина окна:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="pgWidthSpinBox">
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>600</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayoutPGHeight">
        <item row="0" column="1">
         <widget class="QLabel" name="labelPGHeight">
          <property name="text">
           <string>Высота окна:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="pgHeightSpinBox">
          <property name="minimum">
           <number>200</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>800</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayoutPGFPS">
        <item row="0" column="1">
         <widget class="QLabel" name="labelPGFPS">
          <property name="text">
           <string>FPS:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="pmFpsSpinBox">
          <property name="minimum">
           <number>60</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>120</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutRunPg">
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="runPGbutton">
        <property name="text">
         <string>Запустить симуляцию</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutGridButtons">
      <item>
       <widget class="QPushButton" name="addPmObjectButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editPmObjectButton">
        <property name="text">
         <string>Очистить всё</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exportPmObjectButton">
        <property name="text">
         <string>Экспорт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importPmObjectButton">
        <property name="text">
         <string>Импорт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletePmObjectButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="gridPmObjects"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="aboutMenuItem"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aboutMenuItem">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
